src/include/libabrt.h

```diff
@@ -45,6 +45,7 @@ void abrt_ensure_writable_dir(const char *dir, mode_t mode, const char *user);
 void abrt_ensure_writable_dir_group(const char *dir, mode_t mode, const char *user, const char *group);
 char *abrt_run_unstrip_n(const char *dump_dir_name, unsigned timeout_sec);
 char *abrt_get_backtrace(struct dump_dir *dd, unsigned timeout_sec, const char *debuginfo_dirs);
+int abrt_save_coredump(struct dump_dir *dd, unsigned timeout_sec);
```

Comment:
The return value of this function is never checked, nor is it actually needed. Can you please change it to void?

Reply:
Or alternatively, check the return value for errors when you call it. Up to you. The only thing that could fail there is memory allocation, but with g_* familly of functions it will crash the program anyway, so I think void is ok.